components/kafka/src/main/kotlin/org/gxf/servicetemplate/kafka/MeasurementGenerator.kt

```diff
@@ -2,10 +2,10 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-package com.gxf.servicetemplate.kafka
+package org.gxf.servicetemplate.kafka
 
-import com.gxf.service.Measurement
-import com.gxf.service.VoltageMeasurement
+import org.gxf.service.Measurement
+import org.gxf.service.VoltageMeasurement
```

Comment:
VoltageMeasurement is wel erg toegespitst op LSMeten.

Reply:
Het is alleen om als voorbeeld te dienen. Ik wil in deze module niet meer dan 1 objectje.